source/fundamentals/data-formats/serialization.txt

```diff
@@ -173,7 +173,8 @@ package to create a codec for your ``@Serializable`` data classes and
 customize what is stored.
 
 Use the ``BsonConfiguration`` class to define the configuration, 
-including whether to encode defaults, encode nulls, or define class discriminators.
+including whether to encode defaults, encode nulls, define class discriminators,
+or enforce snake case.
```

Comment:
S: could be helpful to link to wikipedia somewhere for anyone unfamiliar with the convention, possibly in an admonition

https://en.wikipedia.org/wiki/Snake_case